.drone.yml

```diff
@@ -1206,20 +1206,23 @@ steps:
   - which curl || apk add --no-cache curl
   - |-
     cd "$WORKSPACE_DIR/go/artifacts"
-    for file in $(find . -type f ! -iname '*.sha256' ! -iname '*-unsigned.zip*'); do
+    find . -type f ! -iname '*.sha256' ! -iname '*-unsigned.zip*' | while read -r file; do
```

Comment:
What's the difference between `for file in ...` vs `find ... | while read ...` - asking for a `bash` deficient person  (i.e. me)

Reply:
Not an expert either, but if I remember correctly: `for` will split by `IFS`, which by default includes spaces. One could in theory use something like `IFS=$'\n' for ...` to only split elements by newline, but I think this did not work as easily with alpine's shell (which does not have some of bash's stuff).

`read X Y Z` will still split the line by word, but if you have more output variables than words, the remainder will end up in the last variable. E.g. with input `foo bar baz quux`, `read X Y Z` will end up in `Z="baz quux"`. Thus, with only one output variable (`read X`), no splitting is performed, and in effect the whole line ends up in `X` unsplitted.